services/ui-src/src/components/reports/PlanOverlayReportPage.tsx

```diff
@@ -0,0 +1,174 @@
+import { useState } from "react";
+// components
+import { Box } from "@chakra-ui/react";
+import {
+  EntityRow,
+  MobileEntityRow,
+  ReportPageFooter,
+  ReportPageIntro,
+  Table,
+} from "components";
+// types
+import { EntityShape, PlanOverlayReportPageShape } from "types";
+// utils
+import { parseCustomHtml, useBreakpoint, useStore } from "utils";
+
+export const PlanOverlayReportPage = ({ route, setSidebarHidden }: Props) => {
```

Comment:
This is great! In terms of the naming, I think we could get ride of `Plan` here in case we end up using this type of page for a different entity type, so either `OverlayReportPage` or `EntityOverlayReportPage` to generalize the page?

Reply:
Bet! I'll go with OverlayReportPage :D